### ISSUE TITLE
Feature request: search

### ISSUE THREAD

**Issue by elimisteve** [cryptag/notes#47]

**Comment by jaduncan** [comment]
This would be really useful to me - I'm likely to pull in just less than a thousand notes, so losing search would be painful.

**Comment by elimisteve** [comment]
@jaduncan Thanks for the feedback; comments like these help me prioritize features!

Where are you pulling in notes from, Google Keep? As I noted in #56, I need to make it easy to import notes from various note-taking apps!

By the way, I'm curious how you heard about CrypTag Notes?

**Comment by jaduncan** [comment]
Yes, Keep. My import process will probably be Takeout followed by a script, but I'm aware that isn't very consumer facing. I can share the script if you like, but it's going to be fairly simple in any case. The biggest blocker for me is attaching the relevant pictures (bug #48, basically).

**Comment by elimisteve** [comment]
@jaduncan Sure seeing your import script would be handy so I can port it to Go and help people import those notes into _CrypTag_ Notes directly from the app itself.

With Keep it looks like the pictures are inlined at the top of the note.  Is this the functionality you need, or would "attaching" photos as if they were email attachments suffice, provided that you could view them when clicked?  Because I plan to add attaching soon, and need to figure out a good way to link to something stored in CrypTag from a note.

...but until then...

If you copy and paste `https://minishare.cryptag.org/#VendorCashewTugboatPantyhoseDubiouslyFanfareSwanIdlyVehicleNevermoreEmoticonCyclist` into CrypTag Notes, like this

![](https://i.giphy.com/OpsEhsvgzGy7C.gif)

then you can see the 2 ways that currently exist to "embed" a picture in a note.

Let me know what would be ideal, thanks!

**Comment by jaduncan** [comment]
I'm afraid that link appears to 404 for me.

**Comment by elimisteve** [comment]
Yes it 404s if you visit it in your browser, but it's an invite link that
you can paste into CrypTag Notes after clicking the "Share" button in the
upper left. Then you'll have access to the folder of notes I've shared with
you so you can see how to add pictures to notes :-).

--Steve

**Comment by jaduncan** [comment]
Yeah, that was my first try. In Notes I get this: "Error fetching invite and creating Backend; error: Error fetching configs: Shares not found for that user"

**Comment by elimisteve** [comment]
@jaduncan Then someone else may have found that link and used it first lol. Sorry for the inconvenience; paste this link into CrypTag Notes instead: `https://minishare.cryptag.org/#PublisherImmigrantSquirrelZeppelinEvolutionRoamerHurricaneWolverineDandelionYesterdayOatPajamas`
